src/InvestmentManager.sol

```diff
@@ -135,7 +135,7 @@ contract InvestmentManager is Auth {
     ///         proceed with tranche token payouts in case their orders got fulfilled.
     /// @dev    The user currency amount required to fulfill the deposit request have to be locked,
     ///         even though the tranche token payout can only happen after epoch execution.
-    function requestDeposit(address liquidityPool, uint256 currencyAmount, address sender, address operator)
+    function requestDeposit(address liquidityPool, uint256 currencyAmount, address receiver, address owner)
```

Comment:
I think these are backwards
```suggestion
    function requestDeposit(address liquidityPool, uint256 currencyAmount, address owner, address receiver)
```

Reply:
should be fine @AStox as the LiquidityPool is calling this function in the following way: 
```
manager.requestDeposit(address(this), assets, receiver, msg.sender), "LiquidityPool/request-deposit-failed"
        );
```

Reply:
Yes I was just making the ordering consistent with `InvestmentManager.requestRedeem` and the ordering of params in `LiquidityPool`.